### DISCUSSION TITLE
How long does GitHub Support take to respond to my tickets?

### DISCUSSION THREAD

**Discussion by BeyondEternalDEV** [community/community#121582] [Other Feature Feedback, Questions, & Ideas] [answered]
I received a suspension on my account due to a third party vulnerability and because of this I have not been able to access it, I opened a ticket over a month ago but I have not received any response, do you know how long it takes for support to respond?

**Comment by gurjeetsinghvirdee** [comment]
Hey @BeyondEternalDEV 
This is a community forum, maintainer's doesn't know about these things,
Many accounts are suspended, so you better make a new account for now

**Comment by ebndev** [comment, marked as answer]
Hi @BeyondEternalDEV,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
